Sources/BandcampDownloader/Business Objects/JSON/JsonTrack.cs

```diff
@@ -13,11 +13,15 @@ internal class JsonTrack {
         [JsonProperty("track_num")]
         public int Number { get; set; }
 
+        [JsonProperty("lyrics")]
```

Comment:
From what I've seen, the `lyrics` property is always empty (for instance with the requested URL in #42), thus I don't think this can work as expected.
Did you manage to make this work?

Reply:
I think I had it working, and verified with some program. There is a chance that the program could have been looking it up though, so I'll re-confirm the functionality.

Reply:
I verified that the lyrics are being tagged properly using [mp3tag](https://www.mp3tag.de/en/) and [this song](https://ripemusic.bandcamp.com/track/brother-sky-2). Not too many programs can utilize it, but the lyrics are stored in the "UNSYNCEDLYRICS" field.

Reply:
Thanks for the confirmation. That's the tag I was aiming to use. 👍 
I've noticed that the lyrics are present on the track pages, but not on the album pages. That's a shame, we'll have to download each track page if we want to download the lyrics. 😞